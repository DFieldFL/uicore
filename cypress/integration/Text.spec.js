/// <reference types="Cypress" />

context('<Text/> Component', () => {
  before(() => {
    cy.visit('http://localhost:3000/text')
    cy.hideStickyHeader()
  })

  after(() => {
    cy.showStickyHeader()
  })

  it('snapshots for Text component', () => {
    cy.get('#text-font-size').toMatchSnapshot()
    cy.get('#text-font-size').toMatchImageSnapshot()

    cy.get('#text-weight').toMatchSnapshot()
    cy.get('#text-weight').toMatchImageSnapshot()

    cy.get('#text-mono').toMatchSnapshot()
    cy.get('#text-mono').toMatchImageSnapshot()

    cy.get('#text-intent').toMatchSnapshot()
    cy.get('#text-intent').toMatchImageSnapshot()

    cy.get('#text-styled-props').toMatchSnapshot()
    cy.get('#text-styled-props').toMatchImageSnapshot()
  })
})
