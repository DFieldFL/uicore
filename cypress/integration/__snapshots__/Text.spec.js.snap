exports[`<Text/> Component > snapshots for Text component #0`] = `
<div
  id="text-font-size"
  class="Layout---vertical--3Szv StyledProps---main--1Wws"
>
  <p class="StyledProps---font--2ipy StyledProps---main--1Wws">
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-small--1qQz"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-normal--3w1_"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-medium--2b-0"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-large--2sHu"
  >
    The quick brown fox jumps over the lazy dog
  </p>
</div>
`;

exports[`<Text/> Component > snapshots for Text component #1`] = `
<div id="text-weight" class="Layout---vertical--3Szv StyledProps---main--1Wws">
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-weight-bold--22XZ"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-weight-semi-bold--QRXr"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-weight-light--2bfm"
  >
    The quick brown fox jumps over the lazy dog
  </p>
</div>
`;

exports[`<Text/> Component > snapshots for Text component #2`] = `
<div id="text-mono" class="Layout---vertical--3Szv StyledProps---main--1Wws">
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---font-mono--1AGe"
  >
    The quick brown fox jumps over the lazy dog
  </p>
</div>
`;

exports[`<Text/> Component > snapshots for Text component #3`] = `
<div id="text-intent" class="Layout---vertical--3Szv StyledProps---main--1Wws">
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---intent-primary--2Xgi"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---intent-success--hn02"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---intent-warning--2N1t"
  >
    The quick brown fox jumps over the lazy dog
  </p>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---intent-danger--1PFK"
  >
    The quick brown fox jumps over the lazy dog
  </p>
</div>
`;

exports[`<Text/> Component > snapshots for Text component #4`] = `
<div
  id="text-styled-props"
  class="Layout---vertical--3Szv StyledProps---main--1Wws"
>
  <p
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---padding-small--3f0A StyledProps---font-size-large--3wvf StyledProps---font-align-center--22lb StyledProps---border--uz_- StyledProps---border-color-green500--1Ccx StyledProps---background-grey100--2FyT"
  >
    The quick brown fox jumps over the lazy dog
  </p>
</div>
`;
